### PR TITLE
[PT] Update Mnesia translation

### PR DESCRIPTION
**Pull request by anderkonzen**
Related to #711

### CODE REVIEW

**Comment by anderkonzen** [comment]
I wasn't sure about what to use for "Match" in "Match and select" section. I ended up using "combinação", but I also thought about using "equivalência" or "correspondência" (as seen in the glossary).

@pragmaticivan, any thoughts here?

**Comment by anderkonzen** [comment]
@ruan-brandao @capaci, guys, can I ask for your thoughts here too?

File: pt/lessons/specifics/mnesia.md [hunk 1]
```diff
@@ -172,3 +179,119 @@ iex> data_to_read = fn ->
...
+...>     {% raw %}Mnesia.select(Person, [{{Person, :"$1", :"$2", :"$3"}, [{:>, :"$1", 3}], [:"$$"]}]){% endraw %}
+...>   end
+...> )
+{:atomic, [[7, "Waylon Smithers", "Executive assistant"], [4, "Marge Simpson", "home maker"], [6, "Monty Burns", "Businessman"], [5, "Hans Moleman", "unknown"]]}
+```
+
+Vamos analisar isto. O primeiro atributo é a tabela, `Person`, e o segundo atributo é uma tripla da forma `{match, [guard], [result]}`:
+
+- `match` é o mesmo que você passaria para a função `Mnesia.match_object/1`. entretanto, note os átomos especiais `:"$n"` que especificam parâmetros posicionais que são usados no restante da consulta
+- a lista `guard` é uma lista de tuplas que especificam qual funções guarda aplicar, neste caso a função integrada `:>` (maior que) com o primeiro parâmetro posicional `:"$1"` e a constante `3` como atributos
```
Comment (ruan-brandao) [inline pt/lessons/specifics/mnesia.md]: "qual _função_ guarda aplicar"

Comment (anderkonzen) [inline pt/lessons/specifics/mnesia.md]: I checked the english version again and I think the following might be better?

EN: "the `guard` list is a list of tuples that specifies what guard functions to apply"
PT: "a lista `guard` é uma lista de tuplas que _especifica_ _quais_ funções guarda aplicar"

Comment (ruan-brandao) [inline pt/lessons/specifics/mnesia.md]: Great! This is better.

Comment (anderkonzen) [inline pt/lessons/specifics/mnesia.md]: Thanks sir! :)

**Comment by anderkonzen** [comment]
@doomspork, are we good to go with this one?

**Comment by doomspork** [comment]
We are good to go @anderkonzen 👍

**Pull request merged**